src/main/kotlin/uk/gov/gdx/datashare/config/S3Config.kt

```diff
@@ -10,7 +10,4 @@ import org.springframework.validation.annotation.Validated
 data class S3Config(
   val localstackUrl: String = "",
```

Comment:
Do we need S3 config at all now?

Reply:
Yeah wasn't sure about this - wasn't sure if it was needed for logs still